palace/models/postoperator.cpp

```diff
@@ -560,16 +562,16 @@ std::complex<double> PostOperator::GetSParameter(const WavePortOperator &wave_po
               "Wave port index " << source_idx << " is not marked for excitation!");
   MFEM_VERIFY(it != wave_port_vi.end(),
               "Could not find wave port when calculating port S-parameters!");
-  std::complex<double> Sij = it->second.S;
+  std::complex<double> S_ij = it->second.S;
   if (idx == source_idx)
   {
-    Sij.real(Sij.real() - 1.0);
+    S_ij.real(S_ij.real() - 1.0);
   }
   // Port de-embedding: S_demb = S exp(ikₙᵢ dᵢ) exp(ikₙⱼ dⱼ) (distance offset is default 0
   // unless specified).
-  Sij *= std::exp(1i * src_data.kn0 * src_data.d_offset);
-  Sij *= std::exp(1i * data.kn0 * data.d_offset);
-  return Sij;
+  S_ij *= std::exp(1i * src_data.kn0 * src_data.d_offset);
+  S_ij *= std::exp(1i * data.kn0 * data.d_offset);
```

Comment:
So the convention (which I agree with), is not snake once within a subscript `_`. Namely, not `S_i_j`.